components/progress-bar/src/overrides/bar.js

```diff
@@ -1,13 +1,14 @@
 /** @jsx jsx */
 
-import { jsx, useBrand } from '@westpac/core';
+import { jsx, useBrand, getLabel } from '@westpac/core';
 
 const Bar = ({ state, ...rest }) => <div {...rest} />;
 
 const barStyles = (_, { look, value }) => {
 	const { COLORS, TYPE } = useBrand();
 
 	return {
+		label: getLabel('progressBar-bar', { look }),
 		position: 'relative',
 		float: 'left',
 		width: `${value}%`,
```

Comment:
well shit... this is not going to work... We have to rethink the entire blender support here!

Reply:
Are you saying we need to provide the value updating feature?

Setting the bar width isnt our concern... same with GUI2 implementation, we just provide the UI

There will be four values that will require updating when this component is used...

- bar width % (css)
- aria-valuenow, 
- aria-valuetext
- visible text value %

Reply:
So users are required to change CSS? That would mean it's not an artifact anymore and can't be generated.

Reply:
```
<div role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="0" aria-valuetext="0% complete" aria-live="polite" class="GEL-progressBar-v1_0_0-beta_2-look_default">
	<div class="GEL-progressBar-v1_0_0-beta_2-bar-look_default" style="width: 0%">
		<span class="GEL-progressbar-v1_0_0-beta_2-text">
			0%
		</span>
	</div>
</div>
```

Why is this not acceptable? `style="width: x%"`

Reply:
yeah totally... That's fine. What we had above was them having to change the css inside the css output which would be bad :D

Reply:
Oh yeah I get you now. Yeah not suggesting they update/edit the css we ship

Reply:
I've updated it to use `style` attribute and removed dependency on the `value` value